src/Connection.php

```diff
@@ -791,7 +800,7 @@ public function executeCommand($name, $params = [])
      * Sends RAW command string to the server.
      * @throws SocketException on connection error.
      */
-    private function sendCommandInternal($command, $params)
+    protected function sendCommandInternal($command, $params)
```

Comment:
```suggestion
    protected function sendCommandRaw($command, $params)
```

It should also be mentioned in the CHANGELOG along with the issue fixed itself.